wagn/lib/decko/engine.rb

```diff
@@ -13,26 +13,36 @@
 require 'diff/lcs'
 require 'diffy'
 
+require 'wagn'
 
 module Decko
+
   class Engine < ::Rails::Engine
 
     paths.add "app/controllers", :with => 'rails/controllers', :eager_load => true
     paths.add 'gem-assets',      :with => 'rails/assets'
     paths.add 'config/routes',   :with => 'rails/engine-routes.rb'
-    paths.add 'lib/tasks',       :with => 'lib/wagn/tasks', :glob => '**/*.rake'
-    
+    paths.add 'lib/tasks',       :with => "#{::Wagn.gem_root}/lib/wagn/tasks", :glob => '**/*.rake'
+    paths.add 'lib/wagn/config/initializers',
+              :with => File.join( Wagn.gem_root, 'lib/wagn/config/initializers' ), :glob => "**/*.rb"
+
+    initializer 'decko.engine.load_config_initializers',  :after => :load_config_initializers do
+      paths['lib/wagn/config/initializers'].existent.sort.each do |initializer|
+        load(initializer)
+      end
+    end
```

Comment:
Is that true?  The tricky part is that we can't depend on Wagn::Application getting defined, because it doesn't.  Some other MyApp < Rails::Application does, and sometimes it does, but these have to load either way.  I would think all that would be defined by this time.  Well, now I'm not sure about :loadOconfig_initializers, need to look for that now.